Project 1/application.py

```diff
@@ -56,35 +56,23 @@ def index():
 
 
 
-# @app.route('/rating',methods = ['GET','POST'])
-# @login_required
-# def rating():
-# 	if(request.method == 'GET'):
-# 		return render_template('rating.html',list = Ratings.query.filter_by(mail = session['name']).all() )
-	
-
-# 	desc,stars = request.form.get('description'), request.form.get('rating')
-# 	db.session.add( Ratings(isbn = '62049879',mail = session['name'],star = stars,description = desc))
-# 	db.session.commit()
-# 	# userlist = Ratings.query.all()
-# 	return render_template('rating.html',list = Ratings.query.filter_by(mail = session['name']).all() )
 
+@app.route('/book/<id>',methods = ['GET','POST'])
```

Comment:
@kamujusrujan @lakshmikiran14  You both are using the same the route for bookpage and review, but rendering different .html pages. Resolve the merge conflicts carefully, while merging them into master.